src/test/java/com/salesforce/storm/spout/dynamic/VirtualSpoutTest.java

```diff
@@ -494,7 +476,7 @@ public void testNextTupleReturnsNullWhenFiltered() {
             null,
             null
         );
-        virtualSpout.getFilterChain().addStep(new SidelineRequestIdentifier("Foobar"), filterStep);
+        virtualSpout.getFilterChain().addStep(new DefaultFilterChainStepIdentifier("Foobar"), filterStep);
```

Comment:
Doesnt look like anything in this test specifically depends on SidelineRequestIdentifier.  Swapped it with the default filter chain step id.